src/main/java/de/retest/recheck/ui/descriptors/AttributeUtil.java

```diff
@@ -51,7 +53,11 @@ public static Rectangle getAbsoluteOutline( final IdentifyingAttributes attribut
 		return outlineAttribute.getValue();
 	}
 
-	private static Rectangle getActualOutline( final ElementDifference difference, final String type ) {
+	protected static Rectangle getActualOutline( final ElementDifference difference, final String type ) {
```

Comment:
Can be package private:

```suggestion
	static Rectangle getActualOutline( final ElementDifference difference, final String type ) {
```